test/core/integration/concrete/lockup-dynamic/create-with-timestamps/createWithTimestamps.t.sol

```diff
@@ -9,20 +10,16 @@ import { stdError } from "forge-std/src/StdError.sol";
 import { ISablierLockupDynamic } from "src/core/interfaces/ISablierLockupDynamic.sol";
 import { Errors } from "src/core/libraries/Errors.sol";
 import { Broker, Lockup, LockupDynamic } from "src/core/types/DataTypes.sol";
-import { CreateWithTimestamps_Integration_Shared_Test } from "./../../../shared/lockup/createWithTimestamps.t.sol";
-import { LockupDynamic_Integration_Concrete_Test } from "./../LockupDynamic.t.sol";
-
-contract CreateWithTimestamps_LockupDynamic_Integration_Concrete_Test is
-    LockupDynamic_Integration_Concrete_Test,
-    CreateWithTimestamps_Integration_Shared_Test
-{
-    function setUp()
-        public
-        virtual
-        override(LockupDynamic_Integration_Concrete_Test, CreateWithTimestamps_Integration_Shared_Test)
-    {
-        LockupDynamic_Integration_Concrete_Test.setUp();
-        CreateWithTimestamps_Integration_Shared_Test.setUp();
+
+import { LockupDynamic_Integration_Shared_Test } from "../LockupDynamic.t.sol";
+
+contract CreateWithTimestamps_LockupDynamic_Integration_Concrete_Test is LockupDynamic_Integration_Shared_Test {
+    uint256 streamId;
+
+    function setUp() public virtual override(LockupDynamic_Integration_Shared_Test) {
```

Comment:
Do we need to mention name of overriding contract here?

Reply:
no, we don't